.travis.yml

```diff
@@ -14,13 +14,17 @@ matrix:
     - python: "2.7"
     - python: "3.4"
     - python: "3.5"
+    - python: "3.6"
+    # Enable 3.7 without globally enabling dist: xenial for other build jobs
+    - python: "3.7"
+      dist: xenial
```

Comment:
Is there any reason to not move everything to `xenial`?

Reply:
Python 3.4 isn't available on Xenial on Travis CI.

One option is `xenial` as default and `trusty` for `3.4`.

Reply:
Going to `xenial` seems like a good idea. We need `sudo` anyway because we need `docker`.

Reply:
Done!

Python 3.4 is EOL in six months anyway, so its exception can be removed soon.